kapitan/refs/secrets/gkms.py

```diff
@@ -93,6 +93,21 @@ def reveal(self):
         ref_data = base64.b64decode(self.data)
         return self._decrypt(ref_data)
 
+    def update_key(self, key):
+        """
+        re-encrypts data with new key, respects original encoding
+        returns True if key is different and secret is updated, False otherwise
+        """
+        if key != self.key:
```

Comment:
It's easier to read if you revers this logic:
```
if key == self.key:
    return False
...
return True
```

Reply:
good point